SEMANTIC_CONVENTIONS.md

```diff
@@ -28,33 +28,106 @@ We use certain data types for the properties we expect.
+| recurrence.interval  | Describing the interval with which the event recurs                                                  | text                                 | OPTIONAL    |
+| tags                 | A list of tags. TODO how to describe?                                                                | list\<text>                          | RECOMMENDED |
+| registration         | If this is a free event, a event which requires registration or a event which requires a paid ticket | enum\<registration>                  | OPTIONAL    |
+| pictureUrl           | Url to a picture to be shown                                                                         | url                                  | DEPRECATED  |
+| pictures.json        | A json array of pictures for the event. The first picture should be the featured one.                | json\<image>                         | OPTIONAL    |
+| collectorName        | Name of the collector which collected this event                                                     | text                                 | MANDATORY   |
+| sources              | A list of all sources, line by line. This should include all URLs or other sources used.             | list\<text>                          | RECOMMENDED |
+| source.event.page    | the url of the page where the event was sourced from                                                 | url                                  | RECOMMENDED |
+| source.event.list    | an url to a list of events on the page where the event was sourced from (if available)               | url                                  | OPTIONAL    |
+| source.event.details | an url that leads to the details of the specific event on the page where the event was sourced from  | url                                  | RECOMMENDED |
+
+#### Usage explanation
+
+`MANDATORY`: This field is required for proper operation of boudicca.
+`RECOMMENDED`: This field is not required, but it is one of the core features enabling event sorting and filtering for publishers.
+`OPTIONAL`: This field can be used if the information is available.
+`DEPRECATED`: This field should no longer be used.
 
 #### Registration enum values
 
 * `free`: a free event which neither requires registration nor a ticket
 * `registration`: an event which requires a free registration
 * `ticket`: a paid event which requires a ticket
 
+#### type
+
+type is an open field, so any value is permitted, but this is a list of common/known types:
+
+- concert
+- theatre
+- meetup
+- museum
+- museum_railway
+- model_railway
+- guided_tour
+- workshop
+- ...
+
 #### Category enum values
 
 * `MUSIC`: concerts or other events where the main focus is music
 * `TECH`: event with technology as the focus
 * `ART`: art exhibitions, comedy, theater, ...
 
+#### RecurrenceType enum values
```

Comment:
i would be fine with those enum values, even though i would not need the distinction between rarely and regularly